warehouse/utils/__init__.py

```diff
@@ -13,8 +13,8 @@
 import datetime
 
 
-def now():
-    return datetime.datetime.utcnow()
+def now() -> datetime.datetime:
+    return datetime.datetime.now(datetime.UTC)
```

Comment:
@miketheman I think this might be causing https://python-software-foundation.sentry.io/issues/4601709612/

Reply:
@di indeed, looks like it. 3.12 deprecates `utcnow()`, which is what prompted this.
Fix incoming!